server/lib/amplitude.js

```diff
@@ -151,7 +152,7 @@ function receiveEvent (event, request, data) {
     )
   );
 
-  if (amplitudeEvent) {
+  if (! amplitude.disabled && amplitudeEvent) {
```

Comment:
Instead of wasting CPU cycles doing the processing only to ignore the result, return early [on line 137](https://github.com/mozilla/fxa-content-server/pull/7088/files#diff-2a83e43540a70e8aa80daa6c26a3de6dR137) if `amplitude.disabled` is true.